athenahealth/enum/appointment_status.go

```diff
@@ -0,0 +1,41 @@
+package enum
+
+import "strconv"
+
+// AppointmentStatus is derived from https://docs.athenahealth.com/api/api-ref/appointment#Book-appointment
```

Comment:
could you add the following to the comment as well
```go
// The athenaNet appointment status. There are several possible statuses.
// x=cancelled
// f=future (It can include appointments where were never checked in, even if the appointment date is in the past. It is up to a practice to cancel appointments as a no show when appropriate to do so.)
// o=open
// 2=checked in
// 3=checked out
// 4=charge entered (i.e. a past appointment)
```